RileyLinkKit/ViewControllers/MySentryPairViewController.swift

```diff
@@ -179,7 +177,7 @@ class MySentryPairViewController: UIViewController, UITextFieldDelegate {
         
         if let data = packet.data, msg = PumpMessage(rxData: data) {
             if msg.packetType == PacketType.MySentry &&
-                msg.address.hexadecimalString == Config.sharedInstance().pumpID {
+                msg.address.hexadecimalString == device.pumpState!.pumpID {
```

Comment:
Is `pumpState` guaranteed to be non-nil to run this view controller? I'm not sure if that's the case today with Naterade.